riff-cli/docs/riff_delete.md

```diff
@@ -23,7 +23,6 @@ riff delete [flags]
 ```
       --all                delete all resources including topics, not just the function resource
       --dry-run            print generated function artifacts content to stdout only
```

Comment:
While we are making changes, maybe this could read "print generated commands to stdout only"